### ISSUE TITLE
[BITV]: create a server side switch for activation / deactivation a "Navigation bar settings" feature

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#42261]
"Navigation bar settings" feature under "Appearance and accessibility" could be not completely accessible. A good decision would be to create a configurable parameter in a config file which can be enabled or disabled for every instance individually.

Please create a comment above this parameter in `config` file with a text: this _feature is not accessible and is not BITV confirm._

Thanks a lot!

**Comment by JuliaKirschenheuter** [comment]
We have decided to keep it. Closing issue

**Comment by szaimen** [comment]
I checked and confirmed that it is still there
